OceanLab/eof.py

```diff
@@ -121,3 +121,162 @@ def my_eof_interp(M,nmodes,errmin=1e-15,repmax=None):
     vi = (M.T+Mmean).T
 
     return vi
+
+#=========================================
+# PERFORM COMPLEX EOF
+#=========================================
+from scipy.signal import hilbert
+import scipy.linalg as la
+from dask import delayed
+from scipy.signal import hilbert
+import xarray as xr
+
+def ceof(lon, lat, data, nkp = 10):
+    ''' Complex (Hilbert) EOF
+    First written in MATLAB and found in Prof. Daniel J. Vimont webpage 
+    (https://www.aos.wisc.edu/~dvimont/matlab/Stat_Tools/complex_eof.html)
+    
+    Translated to Python by Felipe Vilela da Silva @ IMAS, UTas, AU on 15/Mar/2021
+    ==============================================================================
+    Inputs:
+    -> lon: longitudes (array)
```

Comment:
Can you make the inputs following the standards of the other .py? (Maybe it changed after you started this)

It's just capital input/output and the variables below the "U" (3 spaces), with the "=" signed aligned for the larger name:
Example
INPUT:
      Small                = A variable with a small name
      BigNamedOne = A variable with a big name

Reply:
Done